.rubocop.yml

```diff
@@ -1,8 +1,9 @@
 # See full list of defaults here: https://github.com/bbatsov/rubocop/blob/master/config/default.yml
 # To see all cops used see here: https://github.com/bbatsov/rubocop/blob/master/config/enabled.yml
 
-inherit_from:
-  - .rubocop_todo.yml
+require:
+  - rubocop-rails
+  - rubocop-performance
```

Comment:
These were both extracted into separate gems but had previously been running in this repo.